Source/items.cpp

```diff
@@ -1087,7 +1087,7 @@ void SetPlrHandItem(ItemStruct *h, int idata)
 	h->_iPrePower = IPL_INVALID;
 	h->_iSufPower = IPL_INVALID;
 	h->_iMagical = ITEM_QUALITY_NORMAL;
-	h->IDidx = idata;

```

Comment:
Looks like it's only 3 cases where `SetPlrHandItem()` isn't already being used with the enum so why not just type `idata` as `_item_indexes`?

Reply:
I will rename all parameters and variables in all usages once i remake the enum.